### DISCUSSION TITLE
[Kavita] Modify the library for parsing epub

### DISCUSSION THREAD

**Discussion by ghost** [Kareadita/Kavita#3065] [Ideas]
### Idea Description

Can the library for parsing epub be modified? This is too strict for epub. Just having duplicate href in the item will cause an exception and it cannot be displayed in kavita. Many of my epubs do not fully comply with the epub standards, which will cause exceptions. They can be opened in the epub reader and parsed in komga, but this does not work. If they are comics, I can convert them to cbz, but I also have a lot of novels that I cannot manually correct. I can't do anything about them.

### Idea Category

User Experience

### Duration of Using Kavita

Just started

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I can't really do much about bad epubs. I use a downstream library to handle parsing. 

Komga last I heard was pretty strict to confirming to the epub spec and much stricker than Kavita.
